Revert "CB-1817. Add minasshd/GRPC-based CCM service endpoint finder."

sodre90 opened the pull request:
Reverts hortonworks/cloudbreak#5274

Master build fails because of this revert: https://github.com/hortonworks/cloudbreak/pull/5363

As I know we agreed with @yhemanth the cluster-proxy will solve this issue, so we don't need revert SSH, am I wrong? I will drop a mail about this.

jadair-cloudera commented:
I understand this reversion is required by the previous reversion. I am here to comment on your comment about the cluster proxy. Unless there has been a very recent change of direction that was not communicated to the CCM team, CB does not and will not go through the cluster proxy. It talks to the Salt master through the NGINX server, and requires CCM. The FreeIPA management service also does not go through the cluster proxy service, and so also needs . Other CP services do go through the cluster proxy service, and can take advantage of its integration with CCM.

sodre90 commented:
I reverted the previous commit because @daszabo is working on fixing the integration tests (https://github.com/hortonworks/cloudbreak/pull/5339). IT's were disabled since the environment separation commit. The previous commit has fixed ports for knox and gateway (9443, 8443) (https://github.com/hortonworks/cloudbreak/pull/5363/files#diff-4ab6ce4d9d669f56902dcab4dc296fb0L14), but our mock uses random ports, so I reverted the previous change. Can we please revert this commit, then create a PR which contains both commit and has dynamic ports also?
